emailer.py

```diff
@@ -199,7 +199,7 @@ def _get_subject(repo, message):
     else:
         subject_msg = message_lines[0]
     subject_msg = subject_msg[:50]
-    subject = '[{0}] {1}'.format(repo, subject_msg)
+    subject = '[Chapel Merge] {1}'.format(repo, subject_msg)
```

Comment:
Think this needs to be `'[Chapel Merge] {0}'.format(subject_msg)` (drop the repo from the format call, and change `{1}` to `{0}`)

Reply:
D'oh!  Thank you!  Sloppy sloopy sloopppy.